index.d.ts

```diff
@@ -328,7 +328,254 @@ export declare class Stack extends Array<Image> {
   getMinImage(): Image;
 }
 
-export declare class RoiManager {}
```

Comment:
@targos I'm believe I got everything but let me know if you notice an error. Some return types are abstract until we get additional types like `Shape`, etc.